service/src/tests/mocks/mod.rs

```diff
@@ -0,0 +1,23 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+#[cfg(test)]
+pub mod direct_request_mock;
```

Comment:
You don't have to put the `#[cfg(test)]` for every module here. Just put it once for the entire `mocks` module (which is already the case I think) - so it's not needed here.